### PR TITLE
feat(components): add new set of icons

### PR DESCRIPTION
**Pull request by seibei-iguchi**

### CODE REVIEW

File: packages/components/src/components/Icon/Icon.stories.tsx
Comment (jvaclavik) [inline packages/components/src/components/Icon/Icon.stories.tsx:37]: I think `options: allIcons` should be enough

File: packages/components/src/components/Icon/icons.stories.tsx
Comment (jvaclavik) [inline packages/components/src/components/Icon/icons.stories.tsx]: Could we differentiate label for deprecated icons? Like add suffix `(legacy)` or similar. And maybe we could also add radio button to controls with "show all icons / show deprecated icons / show all". WDYT?

File: packages/components/src/components/Icon/Icon.tsx
Comment (jvaclavik) [inline packages/components/src/components/Icon/Icon.tsx:38]: don't we want to load this from shared package? (I'm not sure)

File: packages/components/src/components/Icon/icons.stories.tsx
Comment (jvaclavik) [inline packages/components/src/components/Icon/icons.stories.tsx]: why not just `allIcons.filter(...)`?

File: suite-common/icons/generateIcons.js
Comment (jvaclavik) [inline suite-common/icons/generateIcons.js]: do you need this?

File: packages/components/src/components/Icon/icons.stories.tsx
Comment (seibei-iguchi) [reply packages/components/src/components/Icon/icons.stories.tsx]: Sets don't have filter:)

File: packages/components/src/components/Icon/Icon.tsx
Comment (seibei-iguchi) [reply packages/components/src/components/Icon/Icon.tsx:38]: I don't think so. The shared package contains only the files. I think the sizes should reside with the component, similar to other components.

File: packages/components/src/components/Icon/icons.stories.tsx
Comment (seibei-iguchi) [reply packages/components/src/components/Icon/icons.stories.tsx]: If we add a suffix, we would have to add it to all the currently used icons and we would lose the benefit that some of the names are same and those switched to the new icons automatically.

File: packages/components/src/components/Icon/icons.stories.tsx
Comment (jvaclavik) [reply packages/components/src/components/Icon/icons.stories.tsx]: I see!

**Review by jvaclavik — APPROVED** [review]
Suite desktop: LGTM!

Good job, let's wait for approval from Suite mobile.

**Review by vdovhanych — APPROVED** [review]
Ok for the gh actions part.

**Pull request merged**
